portfolio/src/main/webapp/script.js

```diff
@@ -10,4 +10,15 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+function printMessageFromServer(){
+  fetch("/data")
+  .then(data => data.text())
+  .then(message => {
+    var container = document.getElementById("data-container");
+    container.innerText = message;
+  })
+}
+
+window.onload = () => document.getElementById("data-button").addEventListener("click",printMessageFromServer);
```

Comment:
add a space between `,` and `printMessageFromServer`

Reply:
Updated with commit 95f8b45